app/presenters/image_upload_response_presenter.rb

```diff
@@ -51,6 +51,7 @@ def as_json(*)
       json[:ocr_pii] = ocr_pii
       json[:result_failed] = doc_auth_result_failed?
       json[:doc_type_supported] = doc_type_supported?
+      json[:selfie_status] = selfie_status if show_selfie_failures
```

Comment:
Do we check the mock client `result_response.rb` that implements `show_selfie_failures` as needed?

Reply:
I'm not sure? Can you say more about this?

Reply:
Dawei and I discussed, the mock proofer has changes in other PRs.